pkg/util/process_file_stats.go

```diff
@@ -0,0 +1,11 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package util
+
+type ProcessFileStats struct {
```

Comment:
Would it be possible to add a comment about the purpose of this struct please ? 🙏